### PR TITLE
Fix requests button locators

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/services/test_operations.py --long-running -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 29a8a8a5046ec4264a4b6626ae4a49a33d4ec8a8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/requests.py [hunk 1]
```diff
@@ -10,16 +10,22 @@
...
+        approve={version.LOWEST: "//div[@id='miq_alone' and @title='Approve this Request']/img",
+            "5.5": "//button[@name='miq_request_approve' and @title='Approve this Request']/img"},
+        deny={version.LOWEST: "//div[@id='miq_alone' and @title='Deny this Request']/img",
+            "5.5": "//button[@name='miq_request_deny' and @title='Deny this Request']/img"},
+        copy={version.LOWEST: "//div[@id='miq_alone' and @title='Copy original Request']/img",
+            "5.5": "//button[@name='miq_request_copy' and @title='Copy original Request']/img"},
+        edit={version.LOWEST: "//div[@id='miq_alone' and @title='Edit the original Request']/img",
+            "5.5": "//button[@name='miq_request_edit' and @title='Edit the original Request']/img"},
+        delete={version.LOWEST: "//div[@id='miq_alone' and @title='Delete this Request']/img",
+            "5.5": "//button[@name='miq_request_delete' and @title='Delete this Request']/img"},
```
Comment (mfalesni) [inline cfme/services/requests.py]: I wonder if we could do just like this: `//*[@title='...']/img`, if it is unique, then no verpicking would be required.

Comment (lehinevych) [inline cfme/services/requests.py]: I'll check.

**Comment by dajoRH** [comment]
Lint report for commit 79ccbace5abf6fe6237140852887cd940dba05c4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by lehinevych** [comment]
Failed on 5.4,  sel.click(provisioning_form.submit_copy_button), however manually I couldn't reproduce it, I think the issue with time limit in wait_ajax.

**Comment by mfalesni** [comment]
@lehinevych You might try adding `sel.wait_for_element(provisioning_form.submit_copy_button)` before it.

**Comment by dajoRH** [comment]
Lint report for commit 8acc4088ff21c7346e7b9f2f8e1cb7033a992dd0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by lehinevych** [comment]
@mfalesni thanks, but it didn't help + I couldn't catch this error manually. What am I missing?

**Comment by mfalesni** [comment]
@lehinevych So, I discovered that on 5.4, the button should be only the one that has the `Submit` alt. Not the `Submit the provisioning request`.

**Comment by dajoRH** [comment]
Lint report for commit 9e9052027daea0202d0a922436c339ec0af309ed:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 010c57cc8bf07d1fd684533a7c34937b79f181e3:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/services/test_operations.py:130:8: W292 no newline at end of file
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1f4bc7c6057be3884d97ec98010bb94998f374eb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c56bea81f19120820141101b7facdb7fac8e384b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 927d5cc4bf72cd5f124a2fe3913ae80465318906:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/requests.py [hunk 2]
```diff
@@ -239,7 +240,12 @@ def copy_request(cells):
         raise
     else:
         from cfme.provisioning import provisioning_form
-        sel.click(provisioning_form.submit_copy_button)
```
Comment (psav) [inline cfme/services/requests.py]:
```python
btn = version.pick({
    version.LOWEST: provisioning_form.submit_button,
    '5.5': provisioning_form.submit_copy_button})
sel.wait_for_element(btn)
sel.click(btn)

**Comment by dajoRH** [comment]
Lint report for commit e8988428c58b762c7b5e7199381650a182a22a43:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5ece118aabdb498fc3004b4e55d3f29717835db0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav 
```
TimedOutError
Could not do Any of the elements '(FormButton('Submit this provisioning request'),)' did not appear as expected. at /cfme_tests_te/cfme/fixtures/pytest_selenium.py:272 in time
```

Do we ignore this for 5.4 for now?

**Comment by psav** [comment]
I'd prefer not to since we will be testing 5.4 again soon.

**Comment by lehinevych** [comment]
#2430 the latest run fails on 5.4 however locally it passed. http://pastebin.test.redhat.com/331439

**Comment by dajoRH** [comment]
Lint report for commit 2ae0f7c4db6f8103a6665c460e76abea7afc0227:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
@lehinevych will take a peek on it on subsequent 5.4 runs :smirk:

**Pull request merged**